content/tutorials/custom-zk-chain/10.index.md

```diff
@@ -0,0 +1,264 @@
+```
+
+The `--dev` flag will choose the default options for development. Using this flag is equivalent to answering the prompts like this:
+
+```bash
+◇  Do you want to deploy Paymaster contract?
+│  Yes
+│
+◇  Do you want to deploy some test ERC20s?
+│  Yes
+│
+◇  Do you want to deploy ecosystem contracts? (Not needed if you already have an existing one)
+│  Yes
+│
+◆  What is the RPC URL of the L1 network?
+│  http://localhost:8545 (default)
+```
+
+::callout{icon="i-heroicons-light-bulb"}
+If you have any issues at this step, try reinstalling the dependencies at the top.
+::
+
+This process will take some time as there is a lot happening here.
+To see more detailed logs of what is happening at each step, you can add the `--verbose` flag to the command.
+
+To summarize, the `init` command:
+
+- Checks to see if your environment has the necessary dependencies.
+- Compiles and deploys all of the necessary contracts for the ecosystem.
+- Deploys `zk_chain_1` to the ecosystem.
+- Sets up a database for the default chain (in this case `zk_chain_1`).
+- Deploys a paymaster contract and some test ERC20 contracts (if selected) to use for development.
+
+You can find the paymaster contract deployed to your chain in the `zksync-era` repo in `contracts/l2-contracts/contracts/TestnetPaymaster.sol`,
+and the deployed address inside `my_elastic_chain/chains/zk_chain_1/configs/contracts.yaml` at `l2:testnet_paymaster_addr`.
+
+For the ERC20 contracts, you can find the deployed addresses inside `my_elastic_chain/configs/erc20.yaml`.
+
+### Understanding the chain configs
+
+Running the `init` command will also modify your chain configuration files in the ecosystem folder.
+
+The main configuration file for `zk_chain_1` can be found in `my_elastic_chain/chains/zk_chain_1/ZkStack.yaml`.
+It contains the most basic configurations for the chain.
+
+Inside `my_elastic_chain/chains/zk_chain_1/configs`, you can find six more configuration files:
+
+1. `contracts.yaml`: configurations for all the L1 & L2 contracts.
+1. `external_node.yaml`: configurations for the chain's node server.
+1. `general.yaml`: general configurations.
+1. `genesis.yaml`: chain specific configurations with parameters that were used during genesis.
+1. `secrets.yaml`: secrets that are individual for every chain.
+1. `wallets.yaml`: all wallets that you are using for this chain.
+
+::callout{icon="i-heroicons-light-bulb"}
+Never commit your private keys or sensitive secrets to a public repository!
+::
+
+### Starting the chain server
+
+The last step here is to start a server for `zk_chain_1`:
+
+```bash
+zk_inception server
+```
+
+With this, your L1 chain should be running at port `8545`, and the `zk_chain_1` node should be running at port `3050`.
+
+## Funding a wallet on your chain
+
+Because you chose to use a local reth node for your L1 and selected ETH as the base asset,
+you have access to several rich wallets on the L1 that you can use to bridge ETH to `zk_chain_1`.
```

Comment:
Probably we should mentioned somewhere to never ever use this wallets in production, because they are generated from 1 well known seed phrase